app/src/main/java/com/uc/mymovies/dto/Movie.kt

```diff
@@ -2,6 +2,8 @@ package com.uc.mymovies.dto
 
 data class Movie(val title: String, val genre: String, val director: String, val image: String, val synopsis: String, var isFavorite: Boolean=false) {
 
-
+    override fun toString(): String {
+        return "$title $genre $director $image $synopsis $isFavorite"
+    }
 
```

Comment:
Not necessary, but I think the result can be better formatted

Reply:
Sounds good.